src/scf-release/jobs/acceptance-tests/templates/run.erb

```diff
@@ -48,7 +49,7 @@ fi
 
 set +e
 bin/test -r $noColorFlag -slowSpecThreshold=120 -randomizeAllSpecs $verbose \
-  -nodes=$nodes -skipPackage=helpers $skips -keepGoing "${focus[@]}" \
+  -flakeAttempts=$flakeAttempts -nodes=$nodes -skipPackage=helpers $skips -keepGoing "${focus[@]}" \
```

Comment:
I'd expect shellcheck to complain, but it's not like the rest of the line wouldn't trigger the same…

Reply:
Well, running `shellcheck` against `run.erb` complains about all manner of thing because the file is not a valid shell script.

But once the template is expanded, `shellcheck` will not  complain about these variables because it can now tell that there aren't any spaces in these variable values, so they don't have to be put in quotes.

It also doesn't complain about missing braces because there is no ambiguity in these cases.

I've still fixed the other `shellcheck` errors and added braces everywhere, just for consistency.